news/445.feature.2

```diff
@@ -0,0 +1 @@
+The `env` resolver now attempts to parse the string representation of environment variables in a way that is consistent with the interpolation grammar. This enables more advanced use cases (e.g. setting environment variables that represent lists or dictionaries).
```

Comment:
A slightly simplified version. Maybe a bit less precise but I think it's okay:
```suggestion
The `env` resolver parses environment variables. Supported types includes primitives (int, float, bool etc) and containers like dict and list. The used grammar is a subset of the interpolation grammar.
```

Reply:
Changed in fe57952